UMA-Bots/Large-Relayer-Token-Balance-Change/src/configurables.ts

```diff
@@ -0,0 +1,58 @@
+// @dev the following information is tentative
+
+// minimum percentage (%) change in token balance for a monitored address to trigger an alert
+export const MAINNET_PERCENTAGE_CHANGE_THRESHOLD = 1;
+export const OPTIMISM_PERCENTAGE_CHANGE_THRESHOLD = 10;
+export const POLYGON_PERCENTAGE_CHANGE_THRESHOLD = 10;
+export const ARBITRUM_PERCENTAGE_CHANGE_THRESHOLD = 10;
+
+// List of token contract addresses that are monitored for transfers
+export const MAINNET_MONITORED_TOKENS = [
```

Comment:
If this list is arbitrary, I would suggest adding DAI, (and perhaps UMA, BAL, BOBA) tokens as well as they've got large liquidity in the `HubPool`.